### ISSUE TITLE
Improve precipitation nowcast chart

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#369]
WIP in `better_charts` branch.

Will allow to have feedback before we start using it on more advanced charts.

Already done using AndroidChart:
- Implement the same graph but with the new lib
- Variations are now more accurate (old graph was completely off) using bar charts instead to correctly represent the minute interval
- Make maximum y be at least "heavy rain" level so it’s easier to understand the scale (currently, it's max of the period, so if the max is 0.6 mm/h, it will give the feeling there is a lot of rain where it's 0.6 mm/h, while it's a very light rain).
- Fix times being inaccurate when minute interval is more than 1 minute
- Add thresholds lines for low/moderate/heavy rain to help interpret data (mm/h or in/h is not really easy to understand to most people)

Remains to do:
- Improve marker: make always showing, put correct label, initialize the marker on "current time"
- Define better values for light/medium/heavy precipitation
- Check what's the best way to make thick bars (currently we just put a very high random value)
- More testing
- Wait for Vico v2.x to be stable

**Comment by papjul** [comment]
Vico 2.x to be studied (looks well maintained and modern): https://github.com/patrykandpatrick/vico
